### PR TITLE
Added event attributes with descriptions.

### LINKED ISSUE #1185: Document event attributes for use with filters
Page: https://docs.sensu.io/sensu-go/5.2/reference/filters
To do: 

- Read https://blog.sensu.io/filters-valves-for-the-sensu-monitoring-event-pipeline
- Document attributes to help users better understand the special event data available to use in filters.

Notes:

               "has_check":     e.HasCheck(),
               "has_metrics":   e.HasMetrics(),
               "is_incident":   e.IsIncident(),
               "is_resolution": e.IsResolution(),
               "is_silenced":   e.IsSilenced(),

### PR DESCRIPTION
**Pull request by Blue0ctober**
## Description
Adding Event Attributes for Filters

## Motivation and Context
 Closes #1185

### CODE REVIEW

File: content/sensu-go/5.5/reference/filters.md [hunk 1]
```diff
@@ -320,6 +320,47 @@ example      | {{< highlight shell >}} "annotations": {
   "playbook": "www.example.url"
 }{{< /highlight >}}
 
+### Event attributes
```
Comment (rgeniesse) [inline content/sensu-go/5.5/reference/filters.md:323]: As this is currently marked WIP, possible this is already being done downstream, but I think there needs to be more text here explaining exactly what these event attributes are. That they can be used inside a filter that a user writes and not available inside the context of a handler or mutator execution.

My first read of this, I thought it was talking directly about the built in filters, like `has_metrics` and `is_incident`, which led to confusion about the note and the examples listed.

File: content/sensu-go/5.5/reference/filters.md [hunk 2]
```diff
@@ -320,6 +320,47 @@ example      | {{< highlight shell >}} "annotations": {
...
+type         | String
+example      | {{< highlight shell >}}"has_check": e.IsIncident(){{< /highlight >}}
+
+
+| is_silenced       |      |
+-------------|------
+description  | Determines if an event has any silenced entries.
+required     | false
+type         | String
+example      | {{< highlight shell >}}"has_check": e.IsSilenced(){{< /highlight >}}
```
Comment (rgeniesse) [inline content/sensu-go/5.5/reference/filters.md]:
```suggestion
example      | {{< highlight shell >}}"is_silenced": e.IsSilenced(){{< /highlight >}}
```

**Review by rgeniesse — APPROVED** [review]

**Review by apaskulin — COMMENTED** [review]
I've opened #1419 as a replacement for this PR to address a related issue and make a clear distinction between these attributes and the attributes that are part of the filter spec.

**Comment by apaskulin** [comment]
Closing in favor of #1419

**Pull request closed without merging**